docs: document the Host Rules UI for Mend-hosted Apps (#5)
justo-mend/renovate: docs/usage/mend-hosted/app-secrets.md

   ![Credentials settings page](../assets/images/app-settings/app-credentials.png)

> 4. Use can reference the secret from Renovate config files inside the repo.

Reviewer: ```suggestion
4. Reference the secret from Renovate config files inside the repo.
```

   }
   ```

> ### Adding a host rule

Reviewer: ```suggestion
### Adding a host rule through the UI

You can centrally add/configure Host Rules through the Mend UI as an alternative to including them in Renovate presets.
```

1. Open the _Credentials_ section of the settings page for the relevant Org or Repo.
2. Select `ADD HOST RULE` to open the "Add a Host Rule" dialog box.

   ![Add Host Rule](../assets/images/app-settings/add-host-rule.png)

3. Fill out the details for your host rule.

>    As an example, if you are a Bitbucket or Azure DevOps user, and you want to specify the GITHUB_TOKEN to fetch release notes, you could create a host rule like this:

Reviewer: ```suggestion
   As an example, if you are a Bitbucket or Azure DevOps user, and you want to specify a github.com token to fetch release notes and enable github-based datasources, you could create a host rule like this:
```
